awesome.js

```diff
@@ -278,6 +393,54 @@ class Awesome{
 
         /**
          * loadTemplate collects template element and returns element
+         *
+         * @example
+         *
+         * //taken from awesome-list example, loadTemplate will load template element of awesome-component
+         * //and returns element
+         *
+         * //html snippet
+         *
+         *  <awesome-list>
+        *        <template>
+        *            <li>
+        *                Test 1
+        *            </li>
+        *            <li>
+        *                Test 2
+        *            </li>
+        *            <li>
+        *                Test 3
+        *            </li>
+        *        </template>
+        *    </awesome-list>
+        *
+         * //js
+         *
+         * const content=awesome.loadTemplate(this);
+         *
+         * //constents of content
+        *    `<template>
+        *        <li>
+        *            Test 1
+        *        </li>
+        *        <li>
+        *            Test 2
+        *        </li>
+        *        <li>
+        *            Test 3
+        *        </li>
+        *    </template>`
+         *
```

Comment:
the ***contents*** (sp) of the template do not include the template itself, only its contents

```html


       <li>
           Test 1
       </li>
       <li>
           Test 2
       </li>
       <li>
           Test 3
       </li>

```